hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -122,6 +122,9 @@ public void testCreateTableBuilder() throws Exception {
       assertThat(table.spec().fields()).hasSize(1);
       assertThat(table.properties()).containsEntry("key1", "value1");
       assertThat(table.properties()).containsEntry("key2", "value2");
+      // Default Parquet compression is explicitly set for new tables
+      assertThat(table.properties().get(TableProperties.PARQUET_COMPRESSION))
```

Comment:
minor: `assertThat(table.properties()).containsEntry(TableProperties.PARQUET_COMPRESSION, TableProperties.PARQUET_COMPRESSION_DEFAULT_SINCE_1_4_0)` is usually preferrable, because it will print the content of properties when the assertion ever fails